pages/api/event-feed/index.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from "next";
+import { getEventFeed } from "../../../lib/database/calendar";
+import { StatusCodes } from "http-status-codes";
+
+// handles requests to /api/event-feed/
+const eventFeedHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  switch (req.method) {
```

Comment:
here, `start`, `end` and `userId` will be specified within the query parameters of the request, not the body. Something like this should do the trick:

``` javascript
if (!req.query) {
    return res.status(StatusCodes.INTERNAL_SERVER_ERROR).json("Internal Server Error");
  }

  
  const start = req.query.start as string;
  const end = req.query.end as string;
  const userId = req.query.userId as string;

  if (!start || !end) {
    return res.status(400).json("No start of end date specified");
  }
```